polkadot-parachain/tests/benchmark_storage_works.rs

```diff
@@ -8,7 +8,8 @@ use std::{
 use tempfile::tempdir;
 
 /// The runtimes that this command supports.
-static RUNTIMES: [&str; 3] = ["asset-hub-westend", "asset-hub-kusama", "asset-hub-polkadot"];
+static RUNTIMES: [&str; 4] =
+	["asset-hub-westend", "asset-hub-kusama", "asset-hub-polkadot", "collectives-polkadot"];
```

Comment:
@kalaninja 
why `bridge-hub-*` runtimes are excluded? Because of different `let extra: runtime::SignedExtra` extentions?

what is the purpose or usage of `overhead`?
I just want to understand if we need to add support also for BridgeHub runtimes

Reply:
> what is the purpose or usage of overhead?
they generate base weights for a block and extrinsic. 

> why bridge-hub-* runtimes are excluded?
The reason why I excluded these is that they are excluded in [construct_benchmark_partials](https://github.com/paritytech/cumulus/blob/b894b3d224c64d46bca93f61f34e1daf9e61738d/polkadot-parachain/src/command.rs#L38) macro.

> Because of different let extra: runtime::SignedExtra extentions?
Included runtimes have different SignedExtra too. So, this shouldn't be the reason.

Reply:
@kalaninja 
aha, ok, thank you, 
so let me add them to the `construct_benchmark_partials`

Reply:
@kalaninja 
ok, I added BridgeHubs [here](https://github.com/paritytech/cumulus/pull/2901),
it is merged in master, please, rebase this

Reply:
Thanks @bkontur
I added benchmarks for BridgeHubs. Please, take a look.